cmd/docker/docker.go

```diff
@@ -290,9 +294,15 @@ func runDocker(dockerCli *command.DockerCli) error {
 		return err
 	}
 
+	mp := dockerCli.MeterProvider(ctx)
+	defer mp.Shutdown(ctx)
+	baseOtelAttributes := command.BaseMetricAttributes()
+	stopCmdTimer := command.TimeCommand(ctx, mp, baseOtelAttributes)
```

Comment:
why do we need to call `stopCmdTimer` so many times inside `runDocker`? wouldn't there be a way to move this further up in the call stack and then call this function once inside the `main` function?

Reply:
I also think this is quite ugly. I left a code comment in the main() func regarding this 😛 so we can discuss the approach. 
  
Basically, the `MeterProvider` func currently needs the `dockerCli` struct to be `Initialize`d, and that only happens in here. 

The reason is that the `dockerExporterOtlpEndpoint()` func uses the dockerCli ContextStore to get the endpoint value:
`meta, err := cli.ContextStore().GetMetadata(cli.CurrentContext())`

Reply:
I changed the impl to be much more similar to the compose version you linked. I still think there'd be a good amount of value in having metrics from the very beginning to the very end of the code execution, but we can always handle that in a second moment. 

Let me know what you think 😄